### ISSUE TITLE
Messenger loads messages forever

### ISSUE THREAD

**Issue by ickata** [rsms/fb-mac-messenger#447]
For 2 days the app does not load any message – the progress indicator stays forever. Sign-out/in does not help. Restarting the app does not help. Refreshing also does not help.

**Comment by TomKala** [comment]
i have same problem, still loading 
Mojave 10.14.6
Meseenger 0.1.13

**Comment by timfeid** [comment]
Do you guys have deactivated Facebook accounts?

**Comment by ickata** [comment]
Yes, I do. For years. But I have no problems using Messenger on my iPhone, and I also had no problems using Messenger for Mac until now...

**Comment by timfeid** [comment]
messenger.com does not work with deactivated accounts (since Monday). Facebook needs to fix it.

**Comment by TomKala** [comment]
same situaltion. my facebook is deactivated for 4/5 years. 
im using messenger on iphone for 2 years ( with mobile phone registration).

**Comment by ickata** [comment]
Thanks for the info. Do you have a link to the issue on messenger.com, so we can subscribe to? Thanks!

**Comment by timfeid** [comment]
> Thanks for the info. Do you have a link to the issue on messenger.com, so we can subscribe to? Thanks!

I wish I did, as I am struggling with the same issue. I've submitted a bug report to Facebook, but no response yet.

**Comment by jasdev** [comment]
Seems to be back in action! Messages are loading for me, even with a deactivated account.

**Comment by TomKala** [comment]
yes, now its ok :)))
